src/assertions/descendants.js

```diff
@@ -1,7 +1,7 @@
 export default function descendants ({ wrapper, markup, arg1, sig, flag }) {
   const exactlyCount = flag(this, 'exactlyCount')
 
-  if (exactlyCount) {
```

Comment:
What about null or false?

This should probably be `exactlyCount || exactlyCount === 0`.

Also, it should probably throw on a non-number input, but that can be a separate PR.